internal/resources/ibm/cloudant.go

```diff
@@ -0,0 +1,151 @@
+package ibm
+
+import (
+	"fmt"
+
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+)
+
+const (
+	readCapacity        = 100
+	writeCapacity       = 50
+	globalQueryCapacity = 5
+)
+
+// Resource information: https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/cloudant
+// Pricing information: https://www.ibm.com/cloud/cloudant/pricing
+type Cloudant struct {
+	Address  string
+	Region   string
+	Plan     string
+	Capacity int64
+
+	AdditionalConsumptionStorageGB *int64 `infracost_usage:"additional_consumption_storage_gb"`
+}
+
+// PopulateUsage parses the u schema.UsageData into the Cloudant.
+// It uses the `infracost_usage` struct tags to populate data into the Cloudant.
+func (r *Cloudant) PopulateUsage(u *schema.UsageData) {
+	resources.PopulateArgsWithUsage(r, u)
+}
+
+// CloudantUsageSchema defines a list which represents the usage schema of Cloudant.
+var CloudantUsageSchema = []*schema.UsageItem{
+	{Key: "additional_consumption_storage_gb", DefaultValue: 0},
+}
+
+func (r *Cloudant) BuildResource() *schema.Resource {
+
+	if r.Plan == "lite" {
+		return &schema.Resource{
+			Name:      r.Address,
+			NoPrice:   true,
+			IsSkipped: true,
+		}
+	}
+
+	costComponents := []*schema.CostComponent{
+		r.readCapacityCostComponent(),
+		r.writeCapacityCostComponent(),
+		r.globalQueryCostComponent(),
+		r.additionalCostComponent(),
+	}
+
+	return &schema.Resource{
+		Name:           r.Address,
+		UsageSchema:    CloudantUsageSchema,
+		CostComponents: costComponents,
+	}
+}
+
+func (r *Cloudant) readCapacityCostComponent() *schema.CostComponent {
+	return &schema.CostComponent{
+		Name:            fmt.Sprintf("Read capacity (%d Reads/sec)", r.Capacity*readCapacity),
+		Unit:            "capacity",
+		UnitMultiplier:  decimal.NewFromInt(1),
+		MonthlyQuantity: decimalPtr(decimal.NewFromInt(r.Capacity)),
+		ProductFilter: &schema.ProductFilter{
+			VendorName:    strPtr("ibm"),
+			Region:        strPtr(r.Region),
+			Service:       strPtr("Cloudant"),
```

Comment:
Isn't the service id for cloudant "cloudantnosqldb"? I'm surprised you were able to find a price for this, since I wasn't able to find one in the database under this name?

Reply:
yep, it seems that service is `cloudantnosqldb`.

one example from the `db`. there are 10 different regions:
`['jp-tok', 'au-syd', 'ca-tor', 'br-sao', 'eu-de', 'us-south', 'eu-gb', 'in-che', 'jp-osa', 'us-east']`
```js
{
                "productHash": "31398b25ab5fe98e6f7d01279234e367",
                "vendorName": "ibm",
                "productFamily": "service",
                "region": "jp-tok",
                "service": "cloudantnosqldb"
            },
```

Reply:
@hiltol @sonkatomi 

Since ibm cloud services are not registered with infracost. So we configured self-hosted Cloud Pricing API by adding cloudant service pricing details manually by running sql insert commands.

SQL queries:
https://ibm.box.com/s/96673xjs2u8264xmwt7y94zbyqk12sxf

I am new to cloud-pricing-api. Could you please guide me on how to pull the cloudantnosqldb pricing deatils from cloud pricing API?

Reply:
@anilkumarnagaraj 
1, Clone the forked server:
https://github.com/IBM-Cloud/infracost-cloud-pricing-api
2, Run the catalog scrapers by using `npm run data:scrape:dev`
I would suggest commenting out the AWS/Azure/GCP scrapers
3, Run the service with `npm run dev` (the project has a guide)
4, Configure your CLI to connect to this local instance (the project has a guide)
5, Run queries against the service with the CLI or directly. I would suggest using postman to make graphql queries, its quite convinient.

Reply:
```
query {
  products(filter: {
      vendorName: "ibm",
      service: "cloudantnosqldb",
      region: "us-south"
  }) {
      productHash
      vendorName
      productFamily
      service
      attributes {
        key
        value
      }
      prices {
          startUsageAmount
          endUsageAmount
          priceHash
          unit
          USD
      }
  }
}
```
This will fetch the cloudant data.

Reply:
@balazs-marjan 

Last week I had call with hilton to discuss on configuring "infracost-cloud-pricing-api".

Thanks for your comments.